files/files/translations/locale.pt_BR.properties

```diff
@@ -25,7 +25,7 @@
 # E agora atualizado para a v.1.68 por Fabio Rodolfo Vertamatti
 # Atualizado para v1.73 por Rodrigo Haas
 # Atualizado para v2.21 por Georger Araújo
-# Revisado e atualizado para as versões de 3.0.0 a v4.0.2 por Gérson Vagner (www.gersonvagner.com) 2017/11/25
+# Revisado e atualizado para v4.4.6 por Georger Araújo
```

Comment:
Let's leave a line like `Atualizado para v2.21 por Gérson Vagner (www.gersonvagner.com)`.

Reply:
Yes, that would be fine.